php-7.1/Dockerfile

```diff
@@ -27,12 +27,16 @@ RUN apk add --no-cache \
     php7-bz2 \
     php7-ctype \
     php7-curl \
+    php7-dom \
     php7-iconv \
     php7-json \
     php7-mbstring \
     php7-openssl \
     php7-phar \
     php7-posix \
+    php7-simplexml \
+    php7-tokenizer \
+    php7-xmlwriter \
```

Comment:
I haven't tested these changes* but how come these three extensions are only being added to the 7.1 version and not the 5.6 and 7.0 versions?



(*) see my comment about whether or not they are even worth having

Reply:
They were included in the 'core' in 5.6/7.0 only split out in 7.1

Reply:
👍